### PR TITLE
Added plugin fractalbridge

### PR DESCRIPTION
**Pull request by Popeyef5**

### CODE REVIEW

File: _plugins/fractalbridge.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- automated printing
+- smart queuing
+- fleet management
+
+screenshots:
+- url: /assets/img/plugins/fractalbridge/fractal-home.png
+  alt: Fractal Home
+  caption: Connect your printer to Fractal using OctoPrint
+
+featuredimage: /assets/img/plugins/fractalbridge/fractal-home.jpg
```
Comment (jneilliii) [inline _plugins/fractalbridge.md:34]: incorrect URL for image, should be .png?

**Review by jneilliii — CHANGES_REQUESTED** [review]
one small extension issue commented below, and we strongly recommend having a [software update hook](https://docs.octoprint.org/en/master/bundledplugins/softwareupdate.html#octoprint-plugin-softwareupdate-check-config) so that when you release new versions all users that have it installed get update notices when you publish new releases on GitHub.

File: _plugins/fractalbridge.md (hunk 1 above)
Comment (Popeyef5) [reply _plugins/fractalbridge.md]: Good catch!

**Comment by jneilliii** [comment]
I think you need to make a small tweak to your software update hook implementation. Replace the line [here](https://github.com/FractalManufacturing/OctoPrint-FractalBridge/blob/master/octoprint_fractalbridge/__init__.py#L207) with the following.

```
def __plugin_load__():
    global __plugin_implementation__
    __plugin_implementation__ = FractalBridgePlugin()

    global __plugin_hooks__
    __plugin_hooks__ = {
        "octoprint.plugin.softwareupdate.check_config": __plugin_implementation__.get_update_information,
    }
```

**Comment by Popeyef5** [comment]
Thanks! Done

**Review by jneilliii — APPROVED** [review]
Looks good. Would love to see delta printer support.

**Comment by Popeyef5** [comment]
Thanks! We're launching publicly soon. In principle, it's printer agnostic (i.e. if there's a model you don't see as an option, you can just create it yourself and it will work just fine). We will definitely add Delta printer models as a default option though. We're adding a ton of features, and discussing others here: https://discord.gg/gHjjAqkVsZ

**Comment by jneilliii** [comment]
You may want to give more details to your readme on github, and possibly explain more/ link to fractal site here.

**Comment by Popeyef5** [comment]
Yes, sorry. The initial intent was to make the plugin available for the people beta testing the platform (so that they didn't have to install via a github URL). I didn't know people paid so much attention to plugin releases. I'll be sure to tidy things up in the plugin repo and in the description of the plugin.

**Comment by cp2004** [comment]
Please send in a new PR ASAP to edit the plugin's page on the repository, with at the very least a link to where people can find out more about the service. Or an explanation that the service is not live yet. Something so we don't have to answer endless questions.

**Pull request merged**